.evergreen/config.yml

```diff
@@ -2081,10 +2081,10 @@ axes:
         batchtime: 10080  # 7 days
       - id: rhel8
         display_name: "RHEL 8.x"
-        run_on: rhel87-small
+        run_on: rhel8.8-small
         batchtime: 10080  # 7 days
-      - id: rhel92-fips
-        display_name: "RHEL 9.2 FIPS"
+      - id: rhel9-fips
+        display_name: "RHEL 9 FIPS"
         run_on: rhel92-fips
```

Comment:
```suggestion
        run_on: rhel9-fips
```

Reply:
Does the run_on also need to be changed?

Reply:
No, the host is still rhel92-fips.